stackclub/taster.py

```diff
@@ -137,6 +143,27 @@ def count_things(self):
                 len(self.butler.queryMetadata('src', ['id']))
         return
     
+    def plot_sky_coverage(self):
```

Comment:
I agree that your suggestion is the better way to do it. I tried it this way and for some reason `patches` objects wouldn't appear in the plot, but maybe that was just an error on my end. Since we are only using this method to make a quick plot and the end result will look the same, my inclination is to leave it as is